014/client/src/core/router.js

```diff
@@ -0,0 +1,52 @@
+import MainPage from '../pages/MainPage';
+import SubPage from '../pages/SubPage';
+import DetailPage from '../pages/DetailPage';
+import FavoritePage from '../pages/FavoritePage';
+
+class Router {
+  constructor({ paths }) {
+    this.$pageBody = document.createElement('section');
+    this.$pageBody.className = 'pageBody';
+    this.paths = paths;
+  }
+
+  init() {
+    const $app = document.querySelector('#app');
+    $app.appendChild(this.$pageBody);
+    this.navigate();
+  }
+
+  navigate() {
+    const curPath = window.location.pathname;
+    const isCorrectPath = this.paths.some((path) => path === curPath);
+
+    while (this.$pageBody.hasChildNodes()) {
+      this.$pageBody.removeChild(this.$pageBody.firstChild);
+    }
```

Comment:
```suggestion
    this.$pageBody.children.forEach(v => v.remove());
```
이런식으로 표현할 수 있습니다! 

혹은

```suggestion
    this.$pageBody.innerHTML = '';
```
이렇게 표현해도 괜찮을 것 같아요.